docs/api/features/resources.md

```diff
@@ -1,8 +1,9 @@
-# Resource selection
+# Resource Allocation
```

Comment:
Oh ok. I was wondering about this too and followed "Restart Policies" as an example. Should I change that to "Restart policies"?